src/retrace/retrace.py

```diff
@@ -20,8 +19,6 @@
 from subprocess import *
 import magic
 from .argparser import *
-from webob import Request
```

Comment:
Even though it is not used anywhere in `retrace.py`, it is used in `*.wsgi` scripts through `from retrace import *`.
```
src/backtrace.wsgi:    request = Request(environ)
src/checkpackage.wsgi:    request = Request(environ)
src/create.wsgi:    request = Request(environ)
src/delete.wsgi:    request = Request(environ)
src/exploitable.wsgi:    request = Request(environ)
src/ftp.wsgi:    request = Request(environ)
src/index.wsgi:    request = Request(environ)
src/log.wsgi:    request = Request(environ)
src/manager.wsgi:    request = Request(environ)
src/start.wsgi:    request = Request(environ)
src/stats.wsgi:    request = Request(environ)
src/status.wsgi:    request = Request(environ)
```

Reply:
@mkutlak Thanks for the review. I will update the patch.